app/serializers/api/partnership_serializer.rb

```diff
@@ -0,0 +1,23 @@
+module API
+  class PartnershipSerializer < Blueprinter::Base
+    class AttributesSerializer < Blueprinter::Base
+      exclude :id
+
+      field(:cohort) { |p, _| p.registration_period.year.to_s }
+      field(:urn) { |p, _| p.school.urn.to_s }
+      field(:school_id) { |p, _| p.school.id }
+      field(:delivery_partner_id) { |p, _| p.delivery_partner.id }
+      field(:delivery_partner_name) { |p, _| p.delivery_partner.name }
+
```

Comment:
what about challenged partnerships? we need to show the old information from ECF1 but also we don't support it in the new world? can we add those for now and have that as part of the discussion as well?

Reply:
discussed in person: there are pros/cons to showing challenged vs none. We will be asking providers on this data, and bring up in the next session with RECT